views/default/input/date.php

```diff
@@ -48,8 +52,20 @@
 	$vars['value'] = gmdate('Y-m-d', $vars['value']);
 }
 
+$datepicker_options = elgg_extract('datepicker_options', $vars);
```

Comment:
IMO, an empty string is better, as JS will interpret it as null and set to empty object.